src/components/Header.tsx

```diff
@@ -1,16 +1,135 @@
-import React from 'react';
-import { Navbar, Nav, NavItem, NavLink } from 'reactstrap';
+import React, { useContext } from "react";
+import { HealthContext, SystemContext } from "context";
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  Container,
+  Row,
+  Col,
+  Spinner,
+} from "reactstrap";
+import { parachains } from "config/AvailableParachain";
 
-const Header = () => {
-	return (
-		<Navbar color='light' light expand='md'>
-			<Nav className='mr-auto' navbar>
-				<NavItem>
-					<NavLink href='/components/'>Subspace Relayer</NavLink>
-				</NavItem>
-			</Nav>
-		</Navbar>
-	);
+// TODO: Move
+const bytesToSize = (bytes: number): string => {
+  const sizes = ["Bytes", "KB", "MB", "GB", "TB"];
+  const i = parseFloat(Math.floor(Math.log(bytes) / Math.log(1024)).toString());
+  if (i === 0) return `${bytes} ${sizes[i]}`;
+  return `${(bytes / 1024 ** i).toFixed(1)} ${sizes[i]}`;
+};
+
+const Header = (props: { acumulatedBytes: number; totalBlocks: number }) => {
+  const { version } = useContext(SystemContext);
+  const { isSyncing } = useContext(HealthContext);
+
+  return (
+    <div className="header bg-gradient-gray-dark pb-4 pt-2 pt-md-4 pl-4 pr-9 ">
+      <Container fluid>
+        <Row>
+          <Col lg="2">
+            <Card className="card-stats mb-4 mb-xl-0">
```

Comment:
maybe we can extract `Card` as a separate component and re-use, because it seems like there is some repetition here